tests/test_setup_easytrack_limits.py

```diff
@@ -0,0 +1,140 @@
+def test_permission_pass_for_dai(acl, finance, ldo_holder, amount):
+    assert has_payments_permission(acl, finance, evmscriptexecutor, dai['address'], ldo_holder.address, amount)
+
+    # won't pass here because we don't have dai in treasury
+    with brownie.reverts('VAULT_TOKEN_TRANSFER_REVERTED'):
+        finance.newImmediatePayment(dai['address'], ldo_holder.address, amount, 'test', {'from': evmscriptexecutor})
+
+
+@given(amount=strategy('uint', min_value=eth['limit'] + 1))
+def test_permission_fails_for_eth(acl, finance, ldo_holder, amount):
+    assert not has_payments_permission(acl, finance, evmscriptexecutor, eth['address'], ldo_holder.address, amount)
+    with brownie.reverts('APP_AUTH_FAILED'):
+        finance.newImmediatePayment(eth['address'], ldo_holder.address, amount, 'Should be reverted',
+                                    {'from': evmscriptexecutor})
+
+
+@given(amount=strategy('uint', min_value=steth['limit'] + 1))
+def test_permission_fails_for_steth(acl, finance, ldo_holder, amount):
+    assert not has_payments_permission(acl, finance, evmscriptexecutor, steth['address'], ldo_holder.address, amount)
+    with brownie.reverts('APP_AUTH_FAILED'):
+        finance.newImmediatePayment(steth['address'], ldo_holder.address, amount, 'Should be reverted',
+                                    {'from': evmscriptexecutor})
+
+
+@given(amount=strategy('uint', min_value=ldo['limit'] + 1))
+def test_permission_fails_for_ldo(acl, finance, ldo_holder, amount):
+    assert not has_payments_permission(acl, finance, evmscriptexecutor, ldo['address'], ldo_holder.address, amount)
+    with brownie.reverts('APP_AUTH_FAILED'):
+        finance.newImmediatePayment(ldo['address'], ldo_holder.address, amount, 'Should be reverted',
+                                    {'from': evmscriptexecutor})
+
+
+@given(amount=strategy('uint', min_value=dai['limit'] + 1))
+def test_permission_fails_for_dai(acl, finance, ldo_holder, amount):
+    assert not has_payments_permission(acl, finance, evmscriptexecutor, dai['address'], ldo_holder.address, amount)
+    with brownie.reverts('APP_AUTH_FAILED'):
+        finance.newImmediatePayment(dai['address'], ldo_holder.address, amount, 'Should be reverted',
+                                    {'from': evmscriptexecutor})
+
+
+@given(amount=strategy('uint', min_value=1))
+def test_permission_fails_for_usdc(acl, finance, ldo_holder, amount):
+    assert not has_payments_permission(acl, finance, evmscriptexecutor, usdc_token, ldo_holder.address, amount)
+    with brownie.reverts('APP_AUTH_FAILED'):
+        finance.newImmediatePayment(usdc_token, ldo_holder.address, amount, 'Should be reverted',
+                                    {'from': evmscriptexecutor})
+
+
+@pytest.mark.parametrize('token', [eth, steth, ldo, dai])
+@given(amount=strategy('uint', min_value=1))
+def test_permission_fails_for_other_sender(acl, finance, accounts, token, amount):
```

Comment:
maybe also add another one perm check for another legit sender to show that parameters not applied? I mean `voting` here.